PRINCIPLES.MD

```diff
@@ -0,0 +1,15 @@
+> THIS DOCUMENT IS A DRAFT
+
+# Operating Principles of Fluent UI React Native (FURN)
+- FURN is multi modal – give customers what they want 
+  - Some controls are JS
+  - Some are wrapped-native
+  - Some offer both implementations
+- Cross-platform consistency is an important appeal, but exceptions should be made to account for uniqueness in native platforms – let the platform shines
```

Comment:
# Cross-Platform Considerations
## Consistency
A consistent API and final product is an important synergy for developing with multiple platforms, enabling developers to have a common base of knowledge for each platform..  FURN is not dogmatic in this goal, accounting for the uniqueness in native platform constraints and expected user interaction design.
- The API should align on names and properties whenever appropriate but may favor aligning with the native platform framework over complete consistency in the FURN framework.
- [FluentUI Web](www.github.com/microsoft/fluentui) is an important starting point with its progressive history to avoid reintroducing and re-solving issues that have already been experienced.  Full alignment is not a necessary as the FluentUI Web's React context has incontrovertible technical and contextual differences from the React-Native context that may generate significant, appropriate API and product differences.
- React-Native projections to native platforms can be sorted into the following like categories with significant similarities, driven by device context and user expectations:
- - Desktop OS: Windows, Win32, MacOS
- - Mobile OS: Android, iOS
- - Web

Reply:
A few nits:
- The FluentUI Web seems dead. 
- To be more specific, do we want to align with `@fluent/react` or `@fluentui/react-northstar` ([FAQ for both](https://github.com/microsoft/fluentui/wiki/FAQ---Fabric-and-Stardust-to-Fluent-UI))? I would assume the latter since that's adopting the token pipeline.
- Mac, or macOS, not MacOS =) the capitalization is weird there. I personally prefer the latter.